support/util/scheduler.go

```diff
@@ -0,0 +1,174 @@
+package util
```

Comment:
I think its better to put this file inside controller/scheduler/util package instead of the global util, since its only used there.

Reply:
done